WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/IssueRefundViewModel.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+
+/// ViewModel for presenting the issue refund screen to the user.
+///
+final class IssueRefundViewModel {
+
+    /// Title for the navigation bar
+    /// This is temporary data, will be removed after implementing https://github.com/woocommerce/woocommerce-ios/issues/2842
+    ///
+    let title: String = "$35.45"
+
+    /// String indicating how many items the user has selected to refund
+    /// This is temporary data, will be removed after implementing https://github.com/woocommerce/woocommerce-ios/issues/2842
+    ///
+    let selectedItemsTitle: String = "3 items selected"
+
+    /// The sections and rows to display in the `UITableView`.
+    /// This is temporary data, will be removed after implementing https://github.com/woocommerce/woocommerce-ios/issues/2842
+    ///
+    let sections: [Section] = [
+        Section(rows: [
+            RefundItemViewModel(productImage: nil, productTitle: "Item 1", productQuantityAndPrice: "2 x $30.27 each", quantityToRefund: "1"),
+            RefundItemViewModel(productImage: nil, productTitle: "Item 2", productQuantityAndPrice: "4 x $20.00 each", quantityToRefund: "2"),
+            RefundItemViewModel(productImage: nil, productTitle: "Item 3", productQuantityAndPrice: "3 x $15.99 each", quantityToRefund: "0"),
+            RefundProductsTotalViewModel(productsTax: "$13.45", productsSubtotal: "$66.26", productsTotal: "$79.71")
+        ]),
+        Section(rows: [
+            ShippingSwitchViewModel(title: "Refund Shipping", isOn: true),
+            RefundShippingDetailsViewModel(carrierRate: "USPS Flat Rate",
+                                           carrierCost: "$10.0",
+                                           shippingTax: "$2.99",
+                                           shippingSubtotal: "$10.0",
+                                           shippingTotal: "$12.99")
+        ])
+    ]
+}
+
+// MARK: Sections and Rows
+
+/// Protocol that any `Section` item  should conform to.
+///
+protocol IssueRefundRow {}
+
+extension IssueRefundViewModel {
+
+    struct Section {
+        let rows: [IssueRefundRow]
+    }
+
+    /// ViewModel that represents the shipping switch row.
+    struct ShippingSwitchViewModel: IssueRefundRow {
+        let title: String
+        let isOn: Bool
+    }
+}
+
+extension RefundItemViewModel: IssueRefundRow {}
+
+extension RefundProductsTotalViewModel: IssueRefundRow {}
+
+extension RefundShippingDetailsViewModel: IssueRefundRow {}
```

Comment:
Nothing to change here. Just wanted to mention that I was initially confused about why these were not nested under `IssueRefundViewModel`. 😄 

I'm always concerned about implementing protocols outside of the declaration as they can be difficult to track and pollutes the namespace. I wonder if having a separate nested `struct`, which would clearly state that the `struct` can only be used by `IssueRefundViewModel` and the ViewController, would be better. 

```swift
extension IssueRefundViewModel {
    struct ItemRowViewModel: IssueRefundRow {
        let refundItem: RefundItemViewModel
    }
}
```

It does become repetitive though. 

And again, just thinking out loud for now. I'm open to seeing how this would evolve. 🙂

Reply:
Yeah, I hear you on that the protocol conformance can be difficult to track, but I also didn't want to overcomplicate it too much. As you say, let's see how this evolves.